src/main/java/networkbook/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,63 @@
+package networkbook.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import networkbook.commons.util.ToStringBuilder;
+import networkbook.logic.Messages;
+import networkbook.logic.parser.CliSyntax;
+import networkbook.model.Model;
+import networkbook.model.person.PersonSortComparator;
+
+/**
+ * Finds and lists all persons in network book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts your list of contacts.\n"
+            + "Parameters: "
+            + CliSyntax.PREFIX_SORT_FIELD + " FIELD "
+            + "[" + CliSyntax.PREFIX_SORT_ORDER + " ORDER]\n"
+            + "Example: " + COMMAND_WORD
+            + CliSyntax.PREFIX_SORT_FIELD + " grad "
+            + CliSyntax.PREFIX_SORT_ORDER + " descending";
+
+    private final PersonSortComparator comparator;
+
+    public SortCommand(PersonSortComparator comparator) {
+        this.comparator = comparator;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateSortedPersonList(comparator);
+        return new CommandResult(
+                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
```

Comment:
Do add test case to test the execution of this command!

Reply:
Ok done!